GliaWidgets/Public/GliaError.swift

```diff
@@ -6,4 +6,5 @@ public enum GliaError: Error {
     case callVisualizerEngagementExists
     case configuringDuringEngagementIsNotAllowed
     case clearingVisitorSessionDuringEngagementIsNotAllowed
+    case startEngagementWithNoQueueIds
```

Comment:
I suggest changing it to something like the error above. Starting engagement with no queue ids is not allowed. This error sounds like we are telling them to start the engagement without queue ids.

Reply:
fixed - afbd35fc13f2d73f62a6ad17e2cc0d51c1624063